src/spyglass/lfp/v1/analytic_signal.py

```diff
@@ -0,0 +1,95 @@
+import os
```

Comment:
@xlsun79 could you remove unused imports (os, LFP) and also run Black on these two files?
```python
python -m black /path/to/spyglass/src/spyglass/lfp/v1/analytic_signal.py
python -m black /path/to/spyglass/src/spyglass/lfp/v1/__init__.py
```

Reply:
I did the removal :) could you explain why we wanna run black?

Reply:
Yes! That's the lab standard for code formatting and there is a test for PR's to get merged (lint pull_request) that needs to be successful in order to allow merging. This test makes sure that things are inline with Black formatting